include/os/osEnv.h

```diff
@@ -49,14 +49,20 @@ extern SDiskSpace tsTempSpace;
 void osDefaultInit();
 void osUpdate();
 void osCleanup();
+
 bool osLogSpaceAvailable();
 bool osDataSpaceAvailable();
 bool osTempSpaceAvailable();
+
```

Comment:
We classify disk space conditions into two different scenarios : 1. disk space is in shortage, but not full yet (i.e. sufficiency). 2. disk full, no space left on disk at all (i.e. availability).